### PR TITLE
To eliminate 'racadm_port'.

### PR DESCRIPTION
**Pull request by JuneZhou19**

### CODE REVIEW

File: infrasim/model.py [hunk 1]
```diff
@@ -2146,8 +2145,9 @@ def init(self):
             racadm_obj.set_log_path("/var/log/infrasim/{}/racadm.log".
                                     format(self.__node_name))
 
-            if "racadm_port" in self.__node:
-                racadm_obj.set_port_idrac(self.__node["racadm_port"])
+            if "racadm" in self.__node:
+                if "port" in self.__node["racadm"]:
+                    racadm_obj.set_port_idrac(self.__node["racadm"]["port"])
```
Comment (turtle-fly) [inline infrasim/model.py]: @JuneZhou19 line 2148-2150 is doing the same thing with line 2020.
I prefer to keep 2020, and remove 2148-2050 together with function `set_port_idrac`:

- 2020 is more infrasim model init style to set default value
- while 2148-2150 leverage a function `set_port_idrac`, which is not necessarily needed. You can see serveral `set_xxx` in model, but this kind of "set_xxx" is typical to set attribute to a instance while its definition must be outside of this instance's scope, one example is: `bmc_connection_port` is defined in top level of yml file, it shall be synced to both CBMC and CCompute, so there would be `set_port_qemu_ipmi` in CBMC and CCompute

File: test/functional/test_ipmi_console.py [hunk 2]
```diff
@@ -192,15 +191,15 @@ def setUpClass(cls):
...
+        # Retry SSH to make sure ipmi console started
         for i in range(6):
-           try:
-               time.sleep(10)
-               cls.ssh.connect('127.0.0.1', username='', password='', port=9300)
-           except paramiko.ssh_exception.NoValidConnectionsError, e:
-               if i == 5:
-                   raise e
-               continue
+            try:
```
Comment (turtle-fly) [inline test/functional/test_ipmi_console.py]: Tab in vim:

`:set tabstop=4`, number of visual spaces per TAB
`:set softtabstop=4`, number of spaces in tab when editing
`:set expandtab`, tabs are spaces in further editing
`:retab`, apply current tab setting to existing TAB

File: test/functional/test_node_ports.py [hunk 3]
```diff
@@ -129,7 +129,8 @@ def test_start_node1_then_start_node2_no_conflict_ports(self):
         self.node_info_2['ipmi_console_port'] = 9001
         self.node_info_2['bmc_connection_port'] = 9101
         if 'dell' in self.node_info_2['type']:
-            self.node_info_2['racadm_port'] = 10023
+            self.node_info_2['racadm'] = {}
+            self.node_info_2['racadm']['port'] = 10023
```
Comment (turtle-fly) [inline test/functional/test_node_ports.py]: :+1:

**Comment by JuneZhou19** [comment]
@turtle-fly Agree with you. set_port_idrac removed.

**Comment by XiaowenJiang** [comment]
retest this please

**Pull request merged**